mmd_tools/core/logger.py

```
@@ -0,0 +1,228 @@
+import os
+import sys
+from typing import Optional, Dict, Any
+
+from mmd_tools.core.settings import settings
+
+from .log_formatters import MayaFormatter, CompactFormatter
+
+
+def is_maya_environment() -> bool:
+    """Maya環境かどうかを判定"""
+    try:
```

Ruff (0.12.2)

`maya.cmds` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)